scripts/run_nodes.sh

```diff
@@ -104,3 +106,32 @@ for i in $(seq 0 "$(( N_VALIDATORS + N_NON_VALIDATORS - 1 ))"); do
     "$@" \
     2> $auth.log > /dev/null & \
 done
+
+for i in $(seq "$(( N_VALIDATORS + N_NON_VALIDATORS))" "$(( N_VALIDATORS + N_NON_VALIDATORS - 1 + N_LISTENERES))"); do
+  auth=node-$i
+  account_id=${account_ids[$i]}
+  ./target/release/aleph-node purge-chain --base-path $BASE_PATH/$account_id --chain $BASE_PATH/chainspec.json -y
+  ./target/release/aleph-node \
+    --validator \
+    --chain $BASE_PATH/chainspec.json \
+    --base-path $BASE_PATH/$account_id \
+    --name $auth \
+    --node-type nonvalidator \
```

Comment:
this line together with 115 `--validator` is really confusing

Reply:
removed the node_type